app/components/inputs/Input.js

```diff
@@ -97,6 +98,13 @@ class Input extends React.Component{
               ) : null}
             </div>
           ) : null}
+          {!invalid && value && validMessage ? (
```

Comment:
I'd rather this message to be added at the `PurchaseTickets/Form.js`. 

I am not really a fan of overcomplicating components and as `validMessage` is only used at this place, I think it is worth to not add it to `Input.js` component.